model/src/guild/mod.rs

```diff
@@ -72,23 +74,32 @@ pub struct Guild {
     pub features: Vec<String>,
     pub icon: Option<String>,
     pub id: GuildId,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub joined_at: Option<String>,
-    #[serde(default)]
-    pub large: bool,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub large: Option<bool>,
```

Comment:
This change should be in a separate PR imo, it seems this could fly under the radar if included in this PR and it's a different scope of changes.